packages/main/src/plugin/kubernetes-client.ts

```diff
@@ -45,11 +94,14 @@ export class KubernetesClient {
 
   private kubeConfigWatcher: containerDesktopAPI.FileSystemWatcher | undefined;
 
+  private kubeWatcher: any | undefined;
```

Comment:
do we have a type for this object ?

Reply:
No any is because of the Watch API from the `@kubernetes/client-node`